packages/create-wmr/src/index.js

```diff
@@ -7,10 +7,11 @@ import ora from 'ora';
 import kleur from 'kleur';
 import pkgInstall from 'pkg-install';
 const { getPackageManager, install } = pkgInstall;
-const { dim, bold, cyan } = kleur;
+const { dim, bold, cyan, red } = kleur;
 
 sade('create-wmr [dir]', true)
 	.option('--eslint', 'Set up the Preact ESLint configuration (takes a lot longer)', false)
+	.option('--yarn', 'Set up WMR using Yarn instead of NPM as the package manager', false)
```

Comment:
I wonder - if the idea is to have this work with `yarn init` / `yarn create`, we could auto-detect that:

```js
const preferYarn = /yarn/.test(process.env.NPM_EXECPATH);
```

Reply:
Sure, could do that. I did it as a flag just to mirror what I think most other tools use (I know that's how it is on CRA & Preact CLI) but this is definitely cleaner. I'll change this + a note on the ReadMe